### DISCUSSION TITLE
ZS Extension

### DISCUSSION THREAD

**Discussion by zs-3** [VBAndCs/sVB-Small-Visual-Basic#71] [General]
I Made A Extension For Small Basic In CSharp.
https://github.com/zs-3/ZS
I Will Be Happy If it could be use with SVB

**Comment by VBAndCs** [comment]
Hello Ziyaad Saqlain:
Thanks for your concern.
To make an extension for sVB you must target SmallVisualBasicLibrary.dll instead of SmallBasicLibrary.dll, and if you use LitDev, you must target its version for sVB.

________________________________
From: Ziyaad Saqlain ***@***.***>
Sent: Tuesday, July 9, 2024 8:33 AM
To: VBAndCs/sVB-Small-Visual-Basic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [VBAndCs/sVB-Small-Visual-Basic] ZS Extension (Discussion #71)


I Made A Extension For Small Basic In CSharp.
https://github.com/zs-3/ZS
I Will Be Happy If it could be use with SVB
